sdks/go/test/run_validatesrunner_tests.sh

```diff
@@ -314,7 +314,7 @@ if [[ "$JENKINS" == true ]]; then
       || TEST_EXIT_CODE=$? # don't fail fast here; clean up environment before exiting
 else
   echo ">>> RUNNING $RUNNER VALIDATESRUNNER TESTS"
-  go test -v ./sdks/go/test/integration/... $ARGS \
+  go test -run=TestXLang_Prefix -v ./sdks/go/test/integration/... $ARGS \
```

Comment:
And we probably want to continue run all the tests ;)

Reply:
Fixed